Add a note that help readers avoid problems of can not do 'git push' (#9474)
github/docs: content/github/authenticating-to-github/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository.md

4. Run the following command, replacing `PATH-TO-YOUR-FILE-WITH-SENSITIVE-DATA` with the **path to the file you want to remove, not just its filename**. These arguments will:
    - Force Git to process, but not check out, the entire history of every branch and tag
    - Remove the specified file, as well as any empty commits generated as a result

>     - **Remove some configurations such as the remote url stored in the file `.git/config`**. You may want to back up the config file in advance, so you can restore it when the execution is completed.

Reviewer: ```suggestion
    - Remove some configurations, such as the remote URL, stored in the *.git/config* file. You may want to back up this file in advance for restoration later.
```
